build_latest.sh

```diff
@@ -23,12 +23,11 @@ source ./common_functions.sh
 
 if [ ! -z "$1" ]; then
 	version=$1
-fi
-
-if [ ! -z "$(check_version $version)" ]; then
-	echo "ERROR: Invalid Version"
-	echo "Usage: $0 [8|9]"
-	exit 1
+	if [ ! -z "$(check_version $version)" ]; then
+		echo "ERROR: Invalid Version"
+		echo "Usage: $0 [8|9]"
```

Comment:
Can we support 10 now?  We're starting to build it elsewhere.

Reply:
let's worry about adding 10 later, the main priority right now is to get the job working so that we can actually produce multi arch docker images